openedx/core/djangoapps/site_configuration/helpers.py

```diff
@@ -29,6 +29,15 @@ def get_current_site_configuration():
         return None
 
 
+def get_current_site_configuration_values(default=None):
+    """
+    Returns `SiteConfiguration.site_values` for current site.
+    """
+    default = default if default is not None else {}
+    site_configuration = get_current_site_configuration()
+    return site_configuration.site_values if site_configuration else default
```

Comment:
Is there a way to do this without using a single-line if/else? I don't really like them, they get messy and confusing.

Reply:
I've removed inline if/else.